src/client/cli/cmd/alias.cpp

```diff
@@ -130,9 +141,11 @@ mp::ParseCode cmd::Alias::parse_args(mp::ArgParser* parser)
     }
 
     auto instance = definition.left(colon_pos).toStdString();
+    std::string working_directory = parser->isSet(no_alias_dir_mapping_option) ? "default" : "map";
```

Comment:
Not that this is wrong by any stretch, but the convention throughout the code is to prefer `auto` when it's pretty clear what the type is.